perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -7733,6 +7733,21 @@ protected void initialize() {
 		rights.add(new AttributeRights(-1, Role.SELF, Collections.singletonList(ActionType.READ)));
 		attributes.put(attr, rights);
 
+		//urn:perun:group:attribute-def:virt:autoRegistrationEnabled
+		attr = new AttributeDefinition();
+		attr.setNamespace(AttributesManager.NS_GROUP_ATTR_VIRT);
+		attr.setType(Boolean.class.getName());
+		attr.setFriendlyName("autoRegistrationEnabled");
+		attr.setDisplayName("Automatic registration enabled");
+		attr.setDescription("If true, application for this group will be automatically generated" +
+			" when User becomes member of this group's organization.");
```

Comment:
This isn't exactly true. The member will be offered to join this group. The application will be submitted only if he selects this group.